Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -90,6 +90,24 @@ static u32 EFB_Read(const u32 addr)
 	return var;
 }
 
+static void EFB_Write(u32 data, u32 addr)
+{
+	int x = (addr & 0xfff) >> 2;
+	int y = (addr >> 12) & 0x3ff;
+
+	if (addr & 0x00400000)
+	{
+		g_video_backend->Video_AccessEFB(POKE_Z, x, y, (u32)data);
+		DEBUG_LOG(MEMMAP, "EFB Z Write %08x @ %i, %i", (u32)data, x, y);
+	}
+	else
+	{
+		g_video_backend->Video_AccessEFB(POKE_COLOR, x, y, (u32)data);
```

Comment:
It's a bit out of scope of this PR, but what do you think about merging this two functions into:
g_video_backend->Video_AccessEFB(addr - 0x00400000, data);

btw: The casting to (u32) is redundant.